xgi/drawing/xgi_pylab.py

```diff
@@ -54,29 +56,45 @@ def draw(
     ax : matplotlib.pyplot.axes (default=None)
 
     edge_lc : color (str, dict, or iterable, default='black')
-    Color of the edges (dyadic links and borders of the hyperedges).  If str, use the
-    same color for all edges.  If a dict, must contain (edge_id: color_str) pairs.  If
-    iterable, assume the colors are specified in the same order as the edges are found
-    in H.edges.
+        Color of the edges (dyadic links and borders of the hyperedges).  If str, use the
+        same color for all edges.  If a dict, must contain (edge_id: color_str) pairs.  If
+        iterable, assume the colors are specified in the same order as the edges are found
+        in H.edges.
 
     edge_lw :  float (default=1.5)
-    Line width of edges of order 1 (dyadic links).
+        Line width of edges of order 1 (dyadic links).
+
+    edge_fc : str, 4-tuple, ListedColormap, LinearSegmentedColormap, or dict of 4-tuples or strings
+        Color of hyperedges
 
     node_fc : color (str, dict, or iterable, default='white')
-    Color of the nodes.  If str, use the same color for all nodes.  If a dict, must
-    contain (node_id: color_str) pairs.  If other iterable, assume the colors are
-    specified in the same order as the nodes are found in H.nodes.
+        Color of the nodes.  If str, use the same color for all nodes.  If a dict, must
+        contain (node_id: color_str) pairs.  If other iterable, assume the colors are
+        specified in the same order as the nodes are found in H.nodes.
 
     node_ec : color (dict or str, default='black')
-    Color of node borders.  If str, use the same color for all nodes.  If a dict, must
-    contain (node_id: color_str) pairs.  If other iterable, assume the colors are
-    specified in the same order as the nodes are found in H.nodes.
+        Color of node borders.  If str, use the same color for all nodes.  If a dict, must
+        contain (node_id: color_str) pairs.  If other iterable, assume the colors are
+        specified in the same order as the nodes are found in H.nodes.
 
     node_lw : float (default=1.0)
-    Line width of the node borders in pixels.
+        Line width of the node borders in pixels.
 
     node_size : float (default=0.03)
-    Radius of the nodes in pixels
+        Radius of the nodes in pixels
+
+    **kwargs : optional args
+        alternate default values. Values that can be overwritten are the following:
+        * min_node_size
+        * max_node_size
+        * min_edge_linewidth
+        * max_edge_linewidth
+        * min_node_linewidth
+        * max_node_linewidth
```

Comment:
Nitpick: since the other parameter is called `edge_lw`, this should be `min_edge_lw` and so on.

Reply:
fixed.